app/views/admin/profile_fields/_grouped_profile_fields.html.erb

```diff
@@ -1,28 +1,28 @@
 <% @grouped_profile_fields.each do |group| %>
-  <% group_name = group.name.gsub(/\s+/, "_") %>
+  <% escaped_group_name = group.name.gsub(/\s+|\./, "_") %>
```

Comment:
@djuber defined here, and passed along to the other partials at line 15 and 16 of this file

Reply:
https://github.com/forem/forem/blob/main/app/helpers/admin_helper.rb#L10-L12

I think we have a display helper for a similar transformation, is there value in moving this logic into a helper (and out of the view template?)

```ruby
def escaped_name(group_name)
  group_name.tr('.', '_')
end
```

Reply:
I think moving it to a helper makes sense! I think I'll have to chain `.tr` since I'm replacing both period and whitespace `\s` to underscores.

Reply:
I see -
in that case ignore the tr() suggestion - you're matching two types of patterns and replacing both with a single undercore, `tr(' ', '_')` would generate runs of underscores (not what you intended to do), `gsub` it is!